xds/src/main/java/io/grpc/xds/LookasideLb.java

```diff
@@ -276,6 +426,8 @@ private static ManagedChannel initLbChannel(
 
     @Override
     public void onEndpointChanged(EndpointUpdate endpointUpdate) {
+      channelLogger.log(ChannelLogLevel.DEBUG, "Received an EDS update: '%s'",  endpointUpdate);
+
```

Comment:
For block from line 437 to line 446, any reason to rebuild a list of `DropOverload` by copying elements from one list to another? You should reconsider/rework at least something copied from old implementation, not everything can be directly copied/pasted.

Reply:
After invoking the callback, it just break the loop, but will still call `localityStore.updateDropPercentage` and `localityStore.updateLocalityStore(...)` because it's an update, it will update `localityStore`.